### ISSUE TITLE
Multiwindow / free mode support.

### ISSUE THREAD

**Issue by pazos** [koreader/android-luajit-launcher#238]
Mostly useful for ChromeOS or big tablets. Doesn't make much sense in a phone/e-ink device.

Android docs in https://developer.android.com/guide/topics/ui/multi-window.

Some notes:

1. Resize when the app has a native window to draw. That happens on ChromeOS when we're resizing but not on Android. On android resizing needs to happen in onStart() or when we have focus back, not onConfigurationChanged()
2. Disable fullscreen layout while in multiwindow mode. Also need to take constraints into account to calculate width and height (to avoid draw on top of the menu bar).
3. width and height in multiwindow mode are calculated based on reported data by window.decorView. Current implementations of android.getScreenWidth() and android.getScreenHeight will return bogus values.

I will label as won't fix because I won't but PR are indeed *very* welcome.

**Comment by HoanPiTa** [comment]
I just wanna to read a book in 2 languages in a time.

**Comment by pazos** [comment]
> I just wanna to read a book in 2 languages in a time.

I'm afraid you'll need another program. There're plenty of them in android and most should support multiple instances :)

**Comment by HoanPiTa** [comment]
> > I just wanna to read a book in 2 languages in a time.
> 
> I'm afraid you'll need another program. There're plenty of them in android and most should support multiple instances :)
Yes. I am using Neo reader, moon reader for 2 book in a time. But i like koreader :((

**Comment by beetlejuice16** [comment]
Hello, this is a really important feature for me and I would like to try and contribute with a PR. I sadly have no Android app development experience but I do have experience with Lua, would someone be willing to give me some basic pointers on which parts of the code need to be looked at so I can attempt to work my way through it?

Awesome project btw. Thanks for all your work!
